packages/scaffold/src/partials/w3m-connecting-siwe/index.ts

```diff
@@ -0,0 +1,53 @@
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+
+import styles from './styles.js'
+import { OptionsController, StorageUtil } from '@web3modal/core'
+
+@customElement('w3m-connecting-siwe')
+export class W3mConnectingSiwe extends LitElement {
+  public static override styles = styles
+
+  // -- State & Properties -------------------------------- //
```

Comment:
`// -- State & Properties -------------------------------- //` should be replaced with `// -- Members -------------------------------- //` as everywhere else that uses local vars.

See if this applies in other places where you made properties local vars.